.github/scripts/check_formalities.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+source workflow_context/.github/scripts/ci_helpers.sh
+
+RET=0
+for commit in $(git rev-list HEAD ^origin/$BRANCH); do
+	info "=== Checking commit '$commit'"
+	if git show --format='%P' -s $commit | grep -qF ' '; then
+		err "Pull request should not include merge commits"
+		RET=1
+	fi
+
+	author_name="$(git show -s --format=%aN "$commit")"
+	# Pattern \S\+\s\+\S\+ matches >= 2 names i.e. 3 and more e.g. "John Von
+	# Doe" also match
+	if echo $author_name | grep -q '\S\+\s\+\S\+'; then
+		success "Author name ($author_name) seems OK"
+	# Pattern \S\+ matches single names, typical of nicknamess or handles
```

Comment:
Oh, I dont want to be nitpicking. :-/


```suggestion
	# Pattern \S\+ matches single names, typical of nicknames or handles
```

Same copy paste mistake below.